taskmgr/storage/memory.go

```diff
@@ -14,6 +14,19 @@ type InMemoryDatabase struct {
 	Tasks map[uuid.UUID]*task.Task
 }
 
+func (imd *InMemoryDatabase) GetOne(id string) (*task.Task, error) {
+	UUID, err := uuid.Parse(id)
+	if err != nil {
+		return nil, fmt.Errorf("invalid uuid: %s", id)
+	}
+
+	if _, ok := imd.Tasks[UUID]; !ok {
+		return nil, fmt.Errorf("task with ID: %v not found", UUID)
+	}
+
+	return imd.Tasks[UUID], nil
```

Comment:
since you're calling imd.Tasks[UUID] twice, wouldn't it be cleaner to do something like this?

```suggestion
        val, ok := imd.Tasks[UUID]
	if !ok {
		return nil, fmt.Errorf("task with ID: %v not found", UUID)
	}

	return val, nil
```

Reply:
yup